lib/blueshift/index.js

```diff
@@ -52,32 +52,37 @@ Blueshift.prototype.page = function(page){
   if (this.options.retarget) window.blueshift.retarget();
   var properties = page.properties();
   properties._bsft_source = 'segment.com';
+  properties.customer_id = page.userId();
+  properties.anonymousId = page.anonymousId();
+  properties.category = page.category();
+  properties.name = page.name();
+  removeBlanks(properties);
   window.blueshift.pageload(properties);
 };
 
 /**
- * Identify.
- *
- * @param {Identify} identify
+ * Trait Aliases.
  */
-
-Blueshift.prototype.identify = function(identify){
-  if (!identify.userId()) return this.debug('user id required');
-  var traits = identify.traits({ created: 'created_at' });
-  traits._bsft_source = 'segment.com';
-  window.blueshift.identify(traits);
+var traitAliases = {
+  created: 'created_at',
 };
 
 /**
- * Group.
+ * Identify.
  *
- * @param {Group} group
+ * @param {Identify} identify
  */
 
-Blueshift.prototype.group = function(group){
-  var traits = group.traits({ created: 'created_at' });
+Blueshift.prototype.identify = function(identify){
+  if (!identify.userId() || !identify.anonymousId()) {
+    this.debug('user id required');
```

Comment:
Hey sorry, I think I jumped the gun here.
We do actually want to fire an event even if there's no userId, so I've removed the early return again. 

Should I remove the `!identify.anonymousId()` part of the predicate?
It seems like it would always be present.